### DISCUSSION TITLE
How to display posts?

### DISCUSSION THREAD

**Discussion by jerryhall** [austintoddj/canvas#951] [General]
Ok, all setup and created my first post. Nice experience!

Could you point me in the right direction so that I can include a link to my new blog page? I'm also wondering if the top-nav link could go to an index of sorts with a snippet of all posts on one page (or paginated) and then individual posts on their own page. Should I be looking to create those on my own? And if so do you have any best-practices insofar as layout or naming conventions? 

Thanks again!

**Comment by austintoddj** [comment]
So, if you've got the backend all set up and are now able to create/edit posts, it'll be up to you on how to display them on the frontend. If you want the quick setup, check out the [Frontend](https://github.com/austintoddj/canvas#frontend) section of the Readme and you'll be there in no time. 

Let me know if you have trouble setting it up or if that didn't answer the question.
